File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 3/31/2026, 9:38:55 PM
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        hashed = set()
        for n in nums:
            if n in hashed:
                return True
            hashed.add(n)
        return False

File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 3/31/2026, 9:38:50 PM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> s;
        for(int i =0;i<nums.size();i++){
            if(s.find(nums[i])!=s.end()){
                return true;
            }
            s.insert(nums[i]);
        }
        return false;
    }
};